### PR TITLE
SRVKS-573: Add target utilization docs

### PR DESCRIPTION
**Pull request by abrennan89**
Related Jira: https://issues.redhat.com/browse/SRVKS-573

Applies for OCP 4.6+

Direct preview link: https://deploy-preview-38800--osdocs.netlify.app/openshift-enterprise/latest/serverless/autoscaling/serverless-autoscaling-concurrency.html#serverless-target-utilization_serverless-autoscaling-concurrency

### CODE REVIEW

**Review by markusthoemmes — APPROVED** [review]
LGTM!

File: modules/serverless-concurrency-limits-configure-hard.adoc [hunk 1]
```diff
@@ -24,9 +24,9 @@ spec:
       containerConcurrency: 50
 ----
 +
-The default value is `0`, which means that there is no limit on the number of requests that are permitted to flow into one pod of the service at a time.
+The default value is `0`, which means that there is no limit on the number of requests that are permitted to flow into one replica of the service at a time.
 +
-A value greater than `0` specifies the exact number of requests that are permitted to flow into one pod of the service at a time. This example would enable a hard concurrency limit of 50 requests at a time.
+A value greater than `0` specifies the exact number of requests that are permitted to flow into one replica of the service at a time. This example would enable a hard concurrency limit of 50 requests at a time.
```
Comment (mgencur) [inline modules/serverless-concurrency-limits-configure-hard.adoc:29]: This is a nit but isn't "at a time" redundant when we already mention "concurrency" ? Not sure

File: modules/serverless-target-utilization.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * /serverless/autoscaling/serverless-autoscaling-concurrency.adoc
+
+[id="serverless-target-utilization_{context}"]
+= Concurrency target utilization
+
+This value specifies the percentage of the concurrency limit that is actually targeted by the autoscaler. This is also known as specifying the _hotness_ at which a replica runs, which enables the autoscaler to scale up before the defined hard limit is reached.
+
+For example, if the `containerConcurrency` annotation value is set to 10, and the `targetUtilizationPercentage` value is set to 70 percent, the autoscaler creates a new replica when the average number of concurrent requests across all existing replicas reaches 7. Requests numbered 7 to 10 are still be sent to the existing replicas, but this allows for additional replicas to be started in anticipation of being needed when the `containerConcurrency` annotation limit is reached.
```
Comment (mgencur) [inline modules/serverless-target-utilization.adoc:10]: "are still be sent to" ...should this be "are still to be sent to" ? Sorry if my English is limited here.

**Review by mgencur — COMMENTED** [review]
Hi @abrennan89 , I found just some nits. Otherwise looks good.

File: modules/serverless-target-utilization.adoc (hunk 2 above)
Comment (abrennan89) [reply modules/serverless-target-utilization.adoc:10]: Fixed it, good catch, thanks!

**Comment by abrennan89** [comment]
Thanks for the helpful comments @mgencur 🙂 Make some updates, please take another look

**Comment by mgencur** [comment]
QE Approved. Thanks!

File: modules/serverless-autoscaling-maxscale-kn.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@ You can use the `kn service` command with the `--max-scale` flag to create or mo
```
Comment (mjpytlak) [inline modules/serverless-autoscaling-maxscale-kn.adoc:5]: I suspect you are updating existing content, so if you have time, please comment which assemblies this module is included in.

File: modules/serverless-autoscaling-minscale-kn.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@ You can use the `kn service` command with the `--min-scale` flag to create or mo
```
Comment (mjpytlak) [inline modules/serverless-autoscaling-minscale-kn.adoc:5]: Same as above. If time permits, please comment which assemblies this module is included in.

File: modules/serverless-concurrency-limits-configure-hard.adoc [hunk 5]
```diff
@@ -24,9 +24,9 @@ spec:
       containerConcurrency: 50
```
Comment (mjpytlak) [inline modules/serverless-concurrency-limits-configure-hard.adoc:24]: Same as above. If time permits, please comment which assemblies this module is included in.

File: modules/serverless-target-utilization.adoc [hunk 6]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * /serverless/autoscaling/serverless-autoscaling-concurrency.adoc
+
+[id="serverless-target-utilization_{context}"]
+= Concurrency target utilization
+
+This value specifies the percentage of the concurrency limit that is actually targeted by the autoscaler. This is also known as specifying the _hotness_ at which a replica runs, which enables the autoscaler to scale up before the defined hard limit is reached.
+
+For example, if the `containerConcurrency` annotation value is set to 10, and the `targetUtilizationPercentage` value is set to 70 percent, the autoscaler creates a new replica when the average number of concurrent requests across all existing replicas reaches 7. Requests numbered 7 to 10 are still sent to the existing replicas, but this allows for additional replicas to be started in anticipation of being needed when the `containerConcurrency` annotation limit is reached.
```
Comment (mjpytlak) [inline modules/serverless-target-utilization.adoc:10]:
```suggestion
For example, if the `containerConcurrency` annotation value is set to 10, and the `targetUtilizationPercentage` value is set to 70 percent, the autoscaler creates a new replica when the average number of concurrent requests across all existing replicas reaches 7. Requests numbered 7 to 10 are still sent to the existing replicas, but this lets additional replicas start in anticipation of being required when the `containerConcurrency` annotation limit is reached.
```
Suggesting this in terms of L10N.

**Comment by mjpytlak** [comment]
@abrennan89 A few suggestions, but otherwise LGTM.

**Pull request merged**